uncrustify.cfg

```diff
@@ -40,12 +42,14 @@ nl_namespace_brace		= add
 mod_full_brace_if		= remove
 mod_full_brace_if_chain	= true
 mod_full_brace_do		= remove
-mod_full_brace_while		= remove
+mod_full_brace_while	= remove
 mod_full_brace_for		= remove
-mod_full_brace_if_chain		= true
+mod_full_brace_if_chain	= true
 mod_full_brace_nl		= 3
-mod_full_paren_if_bool		= true
+mod_full_paren_if_bool	= true
+mod_move_case_break		= true
 
+sp_cond_colon				= force
 sp_else_brace				= force
 sp_try_brace				= force
```

Comment:
@evsh:  Why are these three so far to the right?

Reply:
To align them with `sp_before_unnamed_ptr_star` line below.